src/derivative_operators/derivative_operator_functions.jl

```diff
@@ -128,6 +128,20 @@ function *(c::Number, A::DerivativeOperator{T,N,Wind}) where {T,N,Wind}
         A.high_boundary_coefs,coefficients,A.coeff_func)
 end
 
+function *(c::AbstractVector{<:Number}, A::DerivativeOperator{T,N,Wind}) where {T,N,Wind}
+    coefficients = A.coefficients === nothing ? c .* ones(T, A.len) : c .* A.coefficients
```

Comment:
supporting numbers using a `FillVector` would be good, because this would break isbits-ness.

Reply:
I meant for this to be a simple way of multiplying a `DerivativeOperator` by a vector of different coefficients. I'll be adding an assert statement at the beginning to check `c` is the right length and just passing `c` rather than `c .* ones(T, A.len)`.